geoipupdate/test/test_geoip_update.py

```diff
@@ -66,15 +74,13 @@ def test_file_content(self):
     def test_raise_value_error(self, mock_urlretrieve, mock_urlopen):
```

Comment:
Please remove `mock_urlretrieve` . Seems to be dead code.

Reply:
Its not dead code we are mocking the file download . Have a look https://github.com/edx/ecommerce-scripts/pull/139/files#diff-58f2ef0f311b67612cde6a98629b4a34R82